### PR TITLE
JSTL - Always show footer without institution for unregistered users

### PR DESCRIPTION
**Pull request by yj-soh**
Standardises the footer to always show, and hide only the institution in the middle if the user does not have one.
Anyone, review, please.

### CODE REVIEW

**Comment by yj-soh** [comment]
The removed line in `InstructorHomeHTMLPersistenceCheck.html` should not actually be there in and is unchecked in the tests as it gives the same result as the actual page when passed through `convertToStandardHtml()`.

**Comment by MrFool** [comment]
I'll take a look soon :)

File: src/test/resources/pages/InstructorHomeHTMLPersistenceCheck.html [hunk 1]
```diff
@@ -330,4 +330,21 @@ <h4 class="modal-title">
     </div>
     
 
-<div class="tooltip fade top in" style="top: 97px; left: 625px; display: block;"><div class="tooltip-arrow"></div><div class="tooltip-inner">Search for student's name</div></div></body></html>
```
Comment (MrFool) [inline src/test/resources/pages/InstructorHomeHTMLPersistenceCheck.html]: Odd, why is a tooltip there and why is it suddenly removed now? Does it still pass without this?

**Comment by MrFool** [comment]
Can merge since it's dev green, just saw the comment about the convertToStandardHtml() thing

**Pull request merged**